griptape/tasks/prompt_video_generation_task.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from typing import Callable
+
+from attrs import Factory, define, field
+
+from griptape.artifacts import TextArtifact, VideoArtifact
+from griptape.drivers import DreamMachineVideoGenerationDriver
+from griptape.tasks import BaseTask, BaseVideoGenerationTask
+from griptape.utils import J2
+
+
+@define
+class PromptVideoGenerationTask(BaseVideoGenerationTask):
```

Comment:
i dont think these need to be two different classes. the `input` can be different types instead